compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -189,6 +189,9 @@
     UNSAFE_CONVERSION_ATTEMPT("unsafe.conversion.attempt"),
 
     INVALID_LITERAL_FOR_TYPE("invalid.literal.for.type"),
```

Comment:
Are we removing all usage of this with this PR?

Reply:
There's one usage in record binding patterns. #19804 is to take a look and address this as well.